### DISCUSSION TITLE
Can we pack other external fonts with the default font and use them directly?

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#194] [General]
I want to pack some external ttf/otf fonts and want to use them through custom tkinter. is it possible to do?
Secondly, how can I access the default font (roboto) and use it in other non customtkinter widgets?

**Comment by TomSchimansky** [comment]
The roboto font is automatically loaded and can be used on Linux and Windows. How you load external fonts depends on your operating system, there are different methods, just google it for tkinter.
